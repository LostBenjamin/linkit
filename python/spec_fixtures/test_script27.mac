%include symbols
$fade-rate 100
$blink-period 200

[doit]
    <CONFIG-FADE-RATE>=<fade-rate>
    <CONFIG-BLINK-PERIOD> = <blink-period>

