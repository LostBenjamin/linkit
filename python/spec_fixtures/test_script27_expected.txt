72:set:flu
4:set:2,100:cfg:0,200:cfg

